File: app/src/main/java/com/example/choppercharlie/MApplication.java
package com.example.choppercharlie;
import android.app.Application;
import android.content.Context;
import com.secneo.sdk.Helper;
public class MApplication extends Application{
    @Override
    protected void attachBaseContext(Context paramContext) {
        super.attachBaseContext(paramContext);
        Helper.install(MApplication.this);
    }
    //this helps the drone connect but is still not working 100%
    //need to connect to drone
}

File: Mobile/app/src/main/java/com/bitcoders/choppercharlie/subsystems/main/MApplication.kt
package com.bitcoders.choppercharlie.subsystems.main
import com.bitcoders.choppercharlie.subsystems.drone.ConnectionActivity
import android.app.Application
import android.content.Context
import com.secneo.sdk.Helper
class MApplication : Application() {
    override fun attachBaseContext(base: Context?) {
        super.attachBaseContext(base)
        Helper.install(this)
    }
}